src/tribler-common/tribler_common/sentry_reporter/sentry_reporter.py

```diff
@@ -165,6 +167,31 @@ def send(event, post_data, sys_info):
 
             return event
 
+    @staticmethod
+    def send_exception_with_confirmation(exception):
```

Comment:
I'm not sure that SentryReporter class is the proper place for this function.
If you have an idea of a better place, please, let me know.